dialoptions.go

```diff
@@ -221,7 +222,28 @@ func WithInitialWindowSize(s int32) DialOption {
 // and any value smaller than that will be ignored.
 func WithInitialConnWindowSize(s int32) DialOption {
 	return newFuncDialOption(func(o *dialOptions) {
-		o.copts.InitialConnWindowSize = s
+		o.copts.StaticConnWindowSize = s
+		o.copts.BdpEstimationEnabled = false
+	})
+}
+
+// WithStaticStreamWindowSize returns a DialOption which sets the initial
+// stream window size to the value provided and prevents dynamic flow control
+// from adjusting it
+func WithStaticStreamWindowSize(s int32) DialOption {
+	return newFuncDialOption(func(o *dialOptions) {
+		o.copts.StaticWindowSize = s
+		o.copts.BdpEstimationEnabled = true
```

Comment:
I dont think this needs to go in this PR , according to Doug's comment and the issue also , for the 1st PR , we just want to introduce new settings : `WithStaticStreamWindowSize` , `WithStaticConnWindowSize` that does exactly what the old settings do. And do not change anything in the old settings.

The change should happen in internal usage which you have done. 
And the 2nd PR (After 2 releases , as mentioned [here](https://github.com/grpc/grpc-go/issues/7923#issuecomment-2549600066)) should make the `o.copts.BdpEstimationEnabled=true` in `WithInitialConnWindowSize`
```suggestion
		o.copts.BdpEstimationEnabled = false
```

Reply:
Thanks for the clarification. I've made the changes